internal/provider/resource_project.go

```diff
@@ -632,8 +628,8 @@ func resourceProjectCreate(ctx context.Context, d *schema.ResourceData, meta int
 		StorePasswords: types.GetTristateBool(d, "store_password"),
 	}
 
-	if v, ok := d.Get("history_retention_seconds").(int); ok && v >= 0 {
-		projectDef.HistoryRetentionSeconds = pointer(int32(v))
+	if v, ok := d.GetOk("history_retention_seconds"); ok && v.(int) >= 0 {
```

Comment:
**(issue)** Hello! The acceptance test is required to assess the behavior. Thanks!